---
sensor_msgs/JointState qdot1
sensor_msgs/JointState qdot2
std_msgs/Float64MultiArray Q2
std_msgs/Float64 trackingPrecision1
std_msgs/Float64 trackingPrecision2